archive/readme.md

```diff
@@ -38,7 +38,7 @@ Then add the cards into it.
 
 ## How does it work
 
-The getArchiveCards() method creates an <object> element for each archived file. The total number of archived files is based on the manually updated NUMBER_OF_FILES.
+The 'getArchiveCards()' method creates an '<object>' element for each archived file. The total number of archived files is based on the manually updated 'NUMBER_OF_FILES'.
```

Comment:
Hello @rahulsust looks like you used single quotes (' '), you want to use backticks found above the Tab key on a US Keyboard. Markdown interprets the backticks as a code snippet, rendering them like this as an example: `your code here`. Let me know if you have any questions and appreciate you taking care of this 🚀